README.md

```diff
@@ -14,7 +14,10 @@ For any changes, please create a feature branch and open a PR for it when you fe
 
 ## Getting Started
 
-This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. Ruby and Rails versions are specified in the project's `.ruby-version` and `Gemfile`.
+This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. 
+This project uses `ruby 2.3.0` and `rails 5.1.1`
+
```

Comment:
I appreciate the effort and the thoroughness, but I think it would be better to omit specific version numbers. If someone were to change either of them, it's likely that they would forget or not know to modify the version number here. For that information someone can always consult the Gemfile or Gemfile.lock.